e2e/terraform/provision-infra/.gitignore

```diff
@@ -1,4 +1,5 @@
 *.zip
+uploads/
```

Comment:
Is this in addition to `provision-nomad/uploads/`, recently added below? If not, we should remove that line below (and probably move this line into that group) so that we're not adding extra ignored folders we don't mean to.

Reply:
it was moved recently, let me updated it.